util/timer_test.cc

```diff
@@ -281,7 +281,7 @@ TEST_F(TimerTest, AddAfterStartTest) {
   ASSERT_EQ(kIterations, count);
 }
 
-TEST_F(TimerTest, DuplicatedFunTest) {
+TEST_F(TimerTest, AddSameFunNameTest) {
```

Comment:
Nit: 'Fun' leads me to think this name is of something 'fun'. :)

Reply:
changed to `func`.